stregsystem/views.py

```diff
@@ -54,7 +55,10 @@
 
 def __get_news():
     try:
-        return News.objects.filter(stop_date__gte=timezone.now(), pub_date__lte=timezone.now()).get()
+        current_time = timezone.now()
+        news_items = News.objects.filter(stop_date__gte=current_time, pub_date__lte=current_time)
```

Comment:
```suggestion
        news_items = News.objects.filter(stop_date__gte=current_time, pub_date__lte=current_time).order_by('?').first()
```
https://docs.djangoproject.com/en/dev/ref/models/querysets/#django.db.models.query.QuerySet.order_by